gui/src/3D/gui3DManager.ts

```diff
@@ -57,6 +58,37 @@ export class GUI3DManager implements IDisposable {
         return this._utilityLayer;
     }
 
+    /** Gets the scaling for all UI elements owned by this manager */
+    public get controlScaling() {
+        return this._customControlsScale;
+    }
+
+    /** Sets the scaling adjustment for all UI elements owned by this manager */
+    public set controlScaling(newScale: number) {
+        if (this._customControlsScale !== newScale && newScale > 0) {
+            let scaleRatio = newScale / this._customControlsScale;
+            this._customControlsScale = newScale;
+
+            this._rootContainer.children.forEach((control: Control3D) => {
+                control.scaling.scaleInPlace(scaleRatio);
+
+                if (newScale !== 1) {
+                    control._isScaledByManager = true;
+                }
+            });
+        }
+    }
+
+    /** Gets if controls attached to this manager are realistically sized, based on the fact that 1 unit length is 1 meter */
+    public get useRealisticScaling() {
+        return this.controlScaling === 0.1;

```

Comment:
This value ( `0.1`) should be fixed in a constant in this class. It is also possible that this will return true if not set to use realistic scaling, but setting control scaling to 0.1 (0.1 / 1 === 0.1) so maybe a flag would make more sense? Not sure of the real usage here though